tests/clearCache.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { expect, test } from "@playwright/test";
+
+test.use({ storageState: "tests/.auth/user.json" });
+
+test.describe("Clear Cache in profile successfully", () => {
+  test.beforeEach(async ({ page }) => {
+    // Navigate to profile page
+    await page.goto("/");
+    const firstFacilityLink = page
+      .getByRole("link")
+      .filter({ hasText: "View" })
+      .first();
+    await expect(firstFacilityLink).toBeVisible({ timeout: 10000 });
+    await firstFacilityLink.click();
```

Comment:
We can use the facilityId from `facility.setup.ts` like below

```suggestion
    facilityId = getFacilityId();
    await page.goto(`/facility/${facilityId}/overview`);
```